### PR TITLE
add metrics doc

### LINKED ISSUE #6: Add Metrics File

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #6

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  * Giving feedback
+
+## Collections
+
+### Custom Dimensions
+* `cd1` - orientation of split. One of `horizontal` or `vertical`.
+* `cd2` - width of browser window.
+* `cd3` - height of browser window.
+* `cd4`- integer % ratio of split width or height for current split.
+* `cd5`- count of currently open non-split tabs
```
Comment (pdehaan) [inline docs/metrics.md:27]: nit: add space after `cd4` and `cd5` (before the hyphen) for consistency.

File: docs/metrics.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+* `cd8` - total `split-start` events in browser session.
+* `cd9` - total orientation toggles from vertical to horizontal (count should decrement if user toggles back to vertical)
+* `cd10` - max concurrent splits in a browser session.
+* `cd11` - active ticks in browser session (can probably get passed in from [telemetry](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/main-ping.html#info))
+* `cd12` - active ticks in session with a vertically split tab focused
+* `cd13` - active ticks in session with a horizontally split tab focused
+
+### Events
+
+#### `toolbar actions`
```
Comment (pdehaan) [inline docs/metrics.md:39]: `toolbar actions` or `toolbar-actions` (space v hyphen)?

File: docs/metrics.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+ev: {integer count of this event since browser start},
+cd1,
+cd2,
+cd3,
+cd5, // note this should record value AFTER state change
+cd6 // note this should record value AFTER state change
+```
+
+
+### `split-select-actions`
```
Comment (pdehaan) [inline docs/metrics.md:100]: nit: should be an h4?

File: docs/metrics.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+ea: button-click
+el: give-feedback
+cd6,
+cd8,
+cd11,
+cd12,
+cd13
+```
+
+### `split-actions`
```
Comment (pdehaan) [inline docs/metrics.md:123]: should be h4 to match `toolbar-actions` heading above?

File: docs/metrics.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+ea: drag-resize
+el: end
+ev: {integer count of this event since browser start}
+cd1,
+cd2,
+cd3,
+cd4
+```
+
+### `browser-session` actions
```
Comment (pdehaan) [inline docs/metrics.md:138]: nit: h4

File: docs/metrics.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+ea: session-start, //ed. note:  this would be sent on next session start, but maybe we could fire it on session end?
+el: report-card,
+cd7,
+cd8,
+cd9,
+cd10,
+cd11,
+cd12,
+cd13
+
```
Comment (pdehaan) [inline docs/metrics.md:150]: nit: remove empty line before the closing code block.

File: docs/metrics.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+### `split-select-actions`
+_invoked on the split 'new tab' UI_
+```
+ec: split-select-actions,
+ea: select-tab,
+el: {new-tab || open-tab},
+ev: {integer id of index of selected button}
+cd5,
+cd6
+
```
Comment (pdehaan) [inline docs/metrics.md:109]: remove empty line?

**Comment by johngruen** [comment]
todo: change window to viewport
verify that we can segment dimensions

File: docs/metrics.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+ev: {integer count of this event since browser start}
+cd1,
+cd2,
+cd3,
+cd4
+```
+
+### `browser-session` actions
+```
+ec: browser-session,
```
Comment (ianb) [inline docs/metrics.md:140]: This event (I don't think any others?) should be marked at ni/notInteractive

File: docs/metrics.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+  * Splitting and unsplitting
+  * Dragging
+  * Changing orientation
+  * Giving feedback
+
+## Collections
+
+### Custom Dimensions
+* `cd1` - orientation of split. One of `horizontal` or `vertical`.
+* `cd2` - width of browser window.
```
Comment (ianb) [inline docs/metrics.md:24]: - Specify that this is the viewport size
- Ensure we can bucket these sizes in the reports, or else we should bucket them before submitting

**Comment by fzzzy** [comment]
Since this is super old and the comments have not been addressed, I'm just going to merge this and open a new issue to deal with the nits.

**Pull request merged**